src/formatters/default_formatter.erl

```diff
@@ -467,7 +467,7 @@ lay_no_comments(Node, Ctxt) ->
                              Tag =:= mixin;
                              Tag =:= ignore_xref;
                              Tag =:= compile ->
-                        %% We need to convert 2-tuples to arity qualifiers here
+                        %% We need to convert tuples with 2 elements to arity qualifiers here
```

Comment:
nit: 2-element tuples (?)

Reply:
That will trigger Sheldon, that's the reason why I wrote it in full form.